<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_set_db2</class>
 <widget class="QWidget" name="Form_set_db2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库(sqlserver)</string>
  </property>
  <widget class="QLabel" name="label_wtgs_set_db2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>机组:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_wtgs_set_db2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_farm_set_db2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_Button_set_db2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>17</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保  存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_farm_set_db2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>风场:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="query_Button_set_db_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>17</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查  询</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_set_db2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>761</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
